package.json

```diff
@@ -10,6 +10,9 @@
     "start": "node build/server.js",
     "dev": "nodemon src/server.ts",
     "build": "tsc -p .",
+    "db:dockerup": "echo \"🐳✨ Starting docker compose...✨🐳\" && docker compose -f docker/docker-compose-db-dev.yml up -d && echo \"🔄️ Running migrations...🔄️\" && npm run prisma:migrate",
```

Comment:
The script is failing to run the migrations for me the first time. It works on a second run. I think it's trying to run the migrations before the container is fully up and running. It might be better to keep these scripts separate.
First time running `db:dockerup`:
```
🐳✨ Starting docker compose...✨🐳
[+] Building 0.0s (0/0)                             
[+] Running 2/2
 ✔ Network docker_default  Created             0.4s 
 ✔ Container postgres-dev  Started             1.5s 
🔄️ Running migrations...🔄️
```
then `Error: P1001: Can't reach database server at 'localhost':'6500'`
on the second run of `db:dockerup`:
```
🐳✨ Starting docker compose...✨🐳
[+] Building 0.0s (0/0)                             
[+] Running 1/0
 ✔ Container postgres-dev  Running             0.0s 
🔄️ Running migrations...🔄️
```
Notice on the first run postgres-dev is "Started" but on the second run it's "Running".
This happens consistently after running `db:dockerdown` and repeating.

Reply:
I didn't had any problems, since it normally always wait till a command is finished before running an other one.

After running `db:dockerup` did you again run `db:dockerup`  without running `db:dockerdown` in between?
I also made changes to the naming of database, so make sure that is also changed in your URL. 

Lets see if it works like this for @oddpharmacy and @FefesG5

Reply:
If i run it once, the migrations fail. If I run it a second time, it is successful. If I run `db:dockerdown`  the next time I try to run `db:dockerup` always fails, and I need to run it a second time to make the migrations.

Reply:
Hmm that's odd never had that issue. I will create an other command where I combine "npm run db:dockerup" and migrations. I really want it in one command since it is easier to run when testing but also for the developers.

What you can try is run `docker system prune` to make sure everything is cleaned up.

Reply:
Hmmm... pruning seemed to fix it. I generally prune pretty regularly, not sure what would have changed.

Reply:
I made now one command maybe it's more logical. 

It think something was not closed well. Happens to me quite often.